plugin/src/main/kotlin/tanvd/grazi/EnglishChonker.kt

```diff
@@ -0,0 +1,44 @@
+package tanvd.grazi
+
+import com.intellij.openapi.application.ApplicationManager
+import org.jetbrains.org.objectweb.asm.*
+import org.jetbrains.org.objectweb.asm.Opcodes.ASM5
+
+
+class EnglishChonker(cv: ClassVisitor) : ClassVisitor(ASM5, cv) {
+    companion object {
+        fun reloadEnglish() {
+            val stream = GraziPlugin::class.java.getResourceAsStream("/org/languagetool/language/English.class")
+            val reader = ClassReader(stream)
+            val writer = ClassWriter(ClassWriter.COMPUTE_MAXS)
+            val visitor = EnglishChonker(writer)
+            reader.accept(visitor, 0)
+
+            val cls = writer.toByteArray()
+            with(ClassLoader::class.java.getDeclaredMethod("defineClass", String::class.java, ByteArray::class.java, Int::class.java, Int::class.java)) {
+                isAccessible = true
+
+                if (ApplicationManager.getApplication().isUnitTestMode) {
+                    invoke(ClassLoader.getSystemClassLoader(), "org.languagetool.language.English", cls, 0, cls.size)
+                } else {
+                    invoke(GraziPlugin.classLoader, "org.languagetool.language.English", cls, 0, cls.size)
+                }
+            }
+        }
+    }
+
+    override fun visitMethod(access: Int, name: String?, descriptor: String?, signature: String?, exceptions: Array<out String>?): MethodVisitor {
+        val mv = cv.visitMethod(access, name, descriptor, signature, exceptions)
+        return if (name.equals("getChunker")) { GetChunkerMethodVisitor(mv) } else { mv }
```

Comment:
`name == "getChunker"`?

Reply:
And unneeded braces. Also you can use `ifTrue`

Reply:
Done